opencti-platform/opencti-front/src/private/components/settings/groups/Group.tsx

```diff
@@ -467,7 +467,7 @@ const Group = ({ groupData }: { groupData: Group_group$key }) => {
       </Grid>
       <GroupEdition
         groupId={group.id}
-        disabled={!isAllowed}
+        disabled={!isAllowed && isSensitive}
```

Comment:
why this change ?
I would naively think that isAllowed is always true if isSensitive is false (a non-sensitive zone is always granted).

Reply:
because the edit button was disabled for non sensitive Group/Role if user was not allowed (not allowed to modified sensitive Group/Role )